README.md

```diff
@@ -23,6 +23,12 @@
 
 Apache NetBeans is an open source development environment, tooling platform, and application framework.
 
+### Status
+   * TravisCI:
+     * [![Build Status](https://travis-ci.org/apache/incubator-netbeans.svg?branch=master)](https://travis-ci.org/apache/incubator-netbeans)
+   * Jenkins: 
+     * [![Build Status](https://builds.apache.org/view/Incubator%20Projects/job/incubator-netbeans-release/badge/icon)](https://builds.apache.org/view/Incubator%20Projects/job/incubator-netbeans-release/)
```

Comment:
Is the release build the right now to link here? It is started only per request - e.g. it is not classical continuous build. There are two continuous builds at Apache:
* Linux: https://builds.apache.org/job/incubator-netbeans-linux/
* Windows: https://builds.apache.org/job/incubator-netbeans-windows/
For me the Linux build is the best indicator of the state of the repository (it is a build that runs the most tests and they are usually passing). The Windows build would need some care, there is still six persistent errors that would need to be addressed. In any case, I consider these two builds better indicators of the repository status than the `incubator-netbeans-release` job.